src/pocketmine/Player.php

```diff
@@ -832,15 +731,15 @@ public function getInAirTicks(){
 	protected function switchLevel(Level $targetLevel){
 		$oldLevel = $this->level;
 		if(parent::switchLevel($targetLevel)){
-			foreach($this->usedChunks as $index => $d){
+			foreach(array_keys($this->usedChunks) as $index){
```

Comment:
I don't think you should change this. It will cause merge conflicts updating from PocketMine-MP, and function calls are expensive.